### PR TITLE
Import a smart contract into the Nuxt UI project scaffold

### PR DESCRIPTION
**Pull request by ymekuria**
Addresses
#295 

This PR adds the the template Add smart contract into the index.vue of the UI when Nuxt is selected as the UI framework in the CLI.

### CODE REVIEW

**Comment by MartinMinkov** [comment]
I seem to get the following error when using `create-nuxt-app` with Windows:

```
npx create-nuxt-app@latest t9                                                                                                                                                                         create-nuxt-app v5.0.0                                                                                                  ✨  Generating Nuxt.js project in t9                                                                                     ? Project name: t9                                                                                                      ? Programming language: JavaScript                                                                                      ? Package manager: Yarn                                                                                                 ? UI framework: None                                                                                                    ? Template engine: HTML                                                                                                 ? Nuxt.js modules: (Press <space> to select, <a> to toggle all, <i> to invert selection)                                ? Linting tools: (Press <space> to select, <a> to toggle all, <i> to invert selection)                                  ? Testing framework: None                                                                                               ? Rendering mode: Universal (SSR / SSG)                                                                                 ? Deployment target: Server (Node.js hosting)                                                                           ? Development tools: (Press <space> to select, <a> to toggle all, <i> to invert selection)                              ? What is your GitHub username? martin minkov                                                                           ? Version control system: Git                                                                                           Trace: TypeError: Cannot convert undefined or null to object                                                                at Object.apply (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\cna-template\template\nuxt\package.js:36:18)                                                                                                        at Object.loadPackage (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\package.js:21:42)                                                                                                         at Object.load (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\package.js:24:26)                                                                                                                at Object.handler (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\saofile.js:147:34)                                                                                                            at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\sao\lib\runActions.js:103:39              at Array.map (<anonymous>)                                                                                              at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\sao\lib\runActions.js:97:30               at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\majo\dist\majo.cjs.js:58:49               at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\cli.js:56:19 
```

The error seems to be thrown from the CLI. Just wanted to notify that this feature doesn't work on Windows due to the CLI tool itself not working for Windows

**Comment by ymekuria** [comment]
> I seem to get the following error when using `create-nuxt-app` with Windows:
> 
> `npx create-nuxt-app@latest t9 create-nuxt-app v5.0.0 ✨ Generating Nuxt.js project in t9 ? Project name: t9 ? Programming language: JavaScript ? Package manager: Yarn ? UI framework: None ? Template engine: HTML ? Nuxt.js modules: (Press <space> to select, <a> to toggle all, <i> to invert selection) ? Linting tools: (Press <space> to select, <a> to toggle all, <i> to invert selection) ? Testing framework: None ? Rendering mode: Universal (SSR / SSG) ? Deployment target: Server (Node.js hosting) ? Development tools: (Press <space> to select, <a> to toggle all, <i> to invert selection) ? What is your GitHub username? martin minkov ? Version control system: Git Trace: TypeError: Cannot convert undefined or null to object at Object.apply (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\cna-template\template\nuxt\package.js:36:18) at Object.loadPackage (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\package.js:21:42) at Object.load (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\package.js:24:26) at Object.handler (C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\saofile.js:147:34) at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\sao\lib\runActions.js:103:39 at Array.map (<anonymous>) at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\sao\lib\runActions.js:97:30 at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\majo\dist\majo.cjs.js:58:49 at C:\Users\Martin\AppData\Local\npm-cache\_npx\4e867334d6c50eaa\node_modules\create-nuxt-app\lib\cli.js:56:19 `
> 
> The error seems to be thrown from the CLI. Just wanted to notify that this feature doesn't work on Windows due to the CLI tool itself not working for Windows

Did you get the same error when using npm as the package manager?

**Comment by MartinMinkov** [comment]
> @MartinMinkov Did you get the same error when using npm as the package manager?

Yep, something with NPM.

File: src/lib/project.js [hunk 1]
```diff
@@ -526,15 +526,76 @@ function scaffoldNuxt() {
     path.join('ui', 'middleware')
   );
 
-  // Read in the NuxtJS config file and add the middleware.
+  // Read in the NuxtJS config file and add the middleware and vite config.
   const nuxtConfig = fs.readFileSync(path.join('ui', 'nuxt.config.js'), 'utf8');
-  const newNuxtConfig = nuxtConfig.replace(
-    /^}(.*?)$/gm, // Search for the last '}' in the file.
+  let newNuxtConfig = nuxtConfig.replace(
+    'export default {', // Search for the last '}' in the file.
```
Comment (MartinMinkov) [inline src/lib/project.js:532]: Can you update this comment? :D

**Review by MartinMinkov — APPROVED** [review]
One small nit, approving in advance!

**Pull request merged**